source/_docs/modules-plugins-known-issues.md

```diff
@@ -339,6 +339,12 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
 **Solution**: See [Caching: Advanced Topics](/docs/caching-advanced-topics/) for details on how to bypass the platform page cache.
 <hr>
 
+### [Force Login](https://wordpress.org/plugins/wp-force-login/){.external}
+**Issue**: This plugin appends a port number using `$_SERVER['SERVER_PORT']` at the end of the URL when the user logs in to the site.
+
+**Solution**: See [Set SERVER_PORT Corectly](https://pantheon.io/docs/server_name-and-server_port/#set-server_port-correctly).
```

Comment:
Would the customer need to replace any values in the example snippet linked? Which values, and replaced to what?

Reply:
No need to modify the snippet.